src/utils/polygonUtils.cpp

```diff
@@ -1347,4 +1347,56 @@ double PolygonUtils::relativeHammingDistance(const Polygons& poly_a, const Polyg
     return hamming_distance / total_area;
 }
 
+void PolygonUtils::makeCircle(Point mid, coord_t radius, Polygons& ret, float a_step)
```

Comment:
Maybe also use this for other places where we create circles, such as `TreeSupport::drawCircles`?

Reply:
I've made the prime tower and tree support use this function now.

There are also places where we're making an ellipse with a similar technique, but I didn't take the effort to make this function also generate ellipses. It's fine for now. I also didn't want to do that in the tests because then the tests depend on code that's not under test.